api/kyverno/v1/rule_types.go

```diff
@@ -36,6 +37,10 @@ type Rule struct {
 	// +optional
 	ExcludeResources MatchResources `json:"exclude,omitempty" yaml:"exclude,omitempty"`
 
+	// RuleConfig defines rule level config options.
+	// +optional
+	Config RuleConfig `json:"config,omitempty" yaml:"config,omitempty"`
```

Comment:
do we need a `config` object? If it contains a single element, may be best to start with that and extend later if needed.